lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -913,15 +1020,15 @@ def _set_scrape_job_spec(self, event):
                     {"groups": alert_groups}
                 )
 
-    def _set_unit_ip(self, _):
+    def _set_unit_ip(self, _: EventBase):
```

Comment:
`typing.Union[RelationJoinedEvent,RelationChangedEvent,RelationBrokenEvent]` or just `RelationEvent`, probably.

It's not a big deal, especially since it's not used at all, but the type hinting will make language servers more accurate.